### PR TITLE
HLT updates on top of CMSSW_744 - 74X

### PR DESCRIPTION
**Pull request by perrotta**
Updates to the menus dumped in CMSSW:
- Rename some HLT paths
- Adjust a few HLT prescales 
- Integration of DiPFJetAve paths for Jet/MET
- Add prescaled control trigger! HLT_MET60_IsoTrk35_Loose
- Add NoBPTX paths from GRun menu to LowPU menu

### CODE REVIEW

File: HLTrigger/Configuration/python/customizeHLTforMC.py [hunk 1]
```diff
@@ -81,7 +81,8 @@ def customizeHLTforMC(process,_fastSim=False):
       "HLT_Mu28NoFiltersNoVtx_CentralCaloJet40_v*",
       "HLT_Mu23NoFiltersNoVtx_Photon23_CaloIdL_v*",
       "HLT_DoubleMu18NoFiltersNoVtx_v*",
-      "HLT_DoubleMuNoFiltersNoVtx_SaveObjects_v*",
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforMC.py]: Need to keep this for the frozen menus.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by perrotta** [comment]
Added the necessary updates for using GlobalTriggerMenu_L1Menu_Collisions2015_lowPU_v3 menu with the LowPU HLT menu.
To do so I also had to merge #9398, now fully included in this PR
Only the LowPU menu (and marginally the developing menus) got modified: none of the "frozen" menus is.

**Comment by mulhearn** [comment]
+1

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
Frigging DAS

**Comment by Martin-Grunewald** [comment]
please test

**Comment by perrotta** [comment]
Since jenkins tests are stuck anyhow, I added a few further updates to the menu:
- Add Bptx gate for HI muon paths
- Adjust a few HLT prescale factors

**Comment by perrotta** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by perrotta** [comment]
+1

**Comment by Martin-Grunewald** [comment]
Rebased to CMSSW_7_4_5

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by mulhearn** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1
@perrotta 
What is going on here?

**Comment by Martin-Grunewald** [comment]
please test
OK, all over again!

**Comment by Martin-Grunewald** [comment]
@mulhearn
Please sign again!

**Comment by perrotta** [comment]
@Martin-Grunewald : the updates were a few days old in fact, but only noticed now by git

@davidlange6 : please, do not prevent merging because of those two commits:
- 8b6aafa4e1f3ba99f0c5e7f324bb3cc33534874e
- 4df1d66773b1f8de19cbf01f7cb7d0a6a842b50a
As you can see, they only updated the list of paths in the txt files that TSG use to build the menus, but none of the menus was touched, nor anything else relevant for the PR.

**Comment by Martin-Grunewald** [comment]
Which now makes the txt files inconsistent with the menus dumped, right?
Or do the menus already take this into account?

**Comment by perrotta** [comment]
@Martin-Grunewald : fully consistent! As you can see the paths added in the list are still commented out. As I already told you three days ago, they were included in the text files so that we developers could have easily shared them when building the new menus and preparing the next development pull requests, which will be necessarily based on this one.

**Comment by Martin-Grunewald** [comment]
Hmm, I do not see  that:
```
-HLT_Mu17_Mu8_SameSign_v*					# JIRA: CMHSLT-163
-HLT_Mu17_Mu8_SameSign_DPhi_v*					# JIRA: CMHSLT-163
```
are removed now, so regenerated dumps would no longer conatin these paths...

Were the txt files inconsistent before?

**Comment by perrotta** [comment]
Yes, you are right Martin about the removed paths... And also the regenerated dumps of online menu would get rid of the MC_* path which still remained in them.

Anyhow, next dumps will need to contain the new paths anyhow. 

On the other hand, feel free of restoring the old txt files if you don't like them. I have prepared (and I am maintaining up to date) another git branch with the further updates, which we can keep updating:
cms-tsg-storm:hltUpdatesOnTopOf745plusEpsilon_74X

**Comment by Martin-Grunewald** [comment]
OK, fair enough - so let's hope this PR, as is, gets integrated asap.

**Comment by Martin-Grunewald** [comment]
@mulhearn
Could you please sign this - the L1 part has not changed wrt your previous signature.
@davidlange6 
Alternatively, David, since the L1 part has not changed, can you please bypass the L1 signature?

Thanks!

**Comment by Martin-Grunewald** [comment]
@mulhearn , @davidlange6 
Please let me know what is holding this up!

**Pull request merged**